### PR TITLE
Utility scripts

### PR DESCRIPTION
**Pull request by vezzi**
I cleaned up a couple of utility scripts I am using almost everyday:
 - compute_undet_index_stats.py : scripts to extract stats about undetermined reads from statusdb
 - hiseqX_run_times.py : print run times for HiSeqX (for troubleshooting)
 - illumina_indexes.yaml : all illumina indexes in yaml format. not used but I plan to use it and I think it is a useful resource
 - illumina_support_FC.sh : utility script to download all info to be sent to illumina every time they ask to troubleshoot a flowcells 
 - userName2NGIname_finder.py : scripts that associates NGInames to user names and vice-versa

I removed an old script used to track deliveries, no longer needed now with delivery page.

### CODE REVIEW

File: illumina_indexes.yaml [hunk 1]
```diff
@@ -0,0 +1,387 @@
+##yaml file describing indexes for each lib prepr
```
Comment (sylvinite) [inline illumina_indexes.yaml]: It's fine to have this file here. However, I feel that it's a bit unintuitive. Perhaps store it in amsystem or a google doc?

File: update_bioinfo_deliveries.py [hunk 2]
```diff
@@ -1,413 +0,0 @@
-import socket
```
Comment (sylvinite) [inline update_bioinfo_deliveries.py]: Is  this deletion intentional? If so, maybe quota_log.py should be removed as well?

**Comment by sylvinite** [comment]
Looks good. The only general comment I have is that some code sections have an overabundance of blank lines. Makes the code longer with no benefit. Nonetheless, it looks fine as it is.

File: update_bioinfo_deliveries.py (hunk 2 above)
Comment (vezzi) [inline update_bioinfo_deliveries.py]: the all file has been removed

**Comment by vezzi** [comment]
thanks, you can merge then. About the yaml file with the indexes.... I created it with an intention and never really used it, not even in the functions I implemented here. Maybe I will include add something in the next iteration.
Anyhow, others might benefit from a yaml file with all indexes and all illumina lib preps (note that majority of lib preps are not supported by us, so putting it in AM system is a bit strange).

**Pull request merged**